munin/https_dns_proxy.plugin

```diff
@@ -46,24 +46,26 @@ pattern='stat\.c:[0-9]+ ([0-9]+) ([0-9]+) ([0-9]+) ([0-9]+) ([0-9]+) ([0-9]+) ([
 IFS='
 '
 for line in $log_lines; do
-  if [[ ! $line =~ $pattern ]]; then
-    echo "stat regexp did not match with line: $line" >&2
+  if [[ $line =~ $pattern ]]; then
+    stat=("${BASH_REMATCH[@]}")
+  # else
```

Comment:
Drop these commented out lines?

Reply:
I left those lines there intentionally, because they are very good for debugging purpose.